src/main/scala/sbtorgpolicies/rules/rules.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sbtorgpolicies
+import cats.data.{Validated, ValidatedNel}
+import cats.syntax.validated._
+import sbtorgpolicies.exceptions.ValidationException
+
+import scala.util.matching.Regex
+
+package object rules {
+
+  type ValidationResult = ValidatedNel[ValidationException, Unit]
+
+  type ValidationFunction = (String) => ValidationResult
+
+  val emptyValidation: ValidationFunction = _ => Validated.valid((): Unit)
+
+  def requiredStringsValidation(list: List[String]): ValidationFunction = {
+
+    def validateList(content: String, list: List[String])(
+        validateString: (String) => Option[String]): ValidationResult =
+      list.foldLeft(List.empty[String]) {
+        case (accumulated, string) => accumulated ++ validateString(string).toList
+      } match {
```

Comment:
Just FYI using `Foldable#foldMap` : 

```scala
import cats.syntax.foldable._
import cats.instances.list._
list.foldMap(s => validateString(s).toList)
```

Or even better using `FunctorFilter#mapFilter`:

```scala
import cats.syntax.functorFilter._
import cats.instances.list._
list.mapFilter(validateString)
```

The last one is similar as the `flatMap` trick often used (that uses the implicit conversion from `Option` to `GenTraversableOnce` (?))
```scala
list.flatMap(validateString)
```